### PR TITLE
Have the README and config.js match

### PR DESCRIPTION
**Pull request by AlphaT3ch**

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both up- and down-votes are disregarded.
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:* 56774a242a6340fce50827cd7a0ab4100bd115df

File: data/views/layouts/main.hbs [hunk 1]
```diff
@@ -5,13 +5,15 @@
...
-        <style type="text/css" media="all">body {
+        <style type="text/css" media="all">
+	    body {
             margin: 0px;
             padding: 0px;
             background-repeat: no-repeat;
             background-position: top right;
             background-color: white;
             color: black;
+	    background: url('/background.jpg?raw=true');
```
Comment (shalecraig) [inline data/views/layouts/main.hbs]: Please indent this properly.

File: README.md [hunk 2]
```diff
@@ -50,7 +50,8 @@ $ cat config.js
         repo: "botwillacceptanything",
         githubAuth: {
             type: "oauth",
-            token: "YOUR_OAUTH_TOKEN"
+            token: "YOUR_OAUTH_TOKEN",
+            webhookSecret = 'SECRET'
```
Comment (mappum) [inline README.md]: That's not valid JSON

**Comment by dos1** [comment]
:-1:

**Comment by SpenserJ** [comment]
:-1: The background is 1mb.

**Comment by RyanMilb** [comment]
:-1:

**Comment by jedahan** [comment]
:-1: please split the commits into two PRs?

**Comment by avdg** [comment]
That background would look funny at 4k screens

**Comment by mwerner** [comment]
:-1:

**Comment by avdg** [comment]
:-1:

**Comment by dazoe** [comment]
:-1:

**Comment by botwillacceptanything** [comment]
#### :-1: The vote failed. This PR will now be closed. Why don't you try some ideas that don't suck next time, you incredible git?

----
**Tallies:**
:+1:: 1 (12.5%) 
:-1:: 7 (87.5%)

**Pull request closed without merging**